### PR TITLE
feat: backdrop stage abstraction

### PR DESCRIPTION
**Pull request by alvarosabu**

### CODE REVIEW

File: public/models/suzanne.json
Comment (JaimeTorrealba) [inline public/models/suzanne.json:1]: Why do we add models and textures here in cientos? 

Should not go in assets repo?

File: docs/public/card-bo-bot/CardBoBot-height.png
Comment (JaimeTorrealba) [inline docs/public/card-bo-bot/CardBoBot-height.png:1]: Same here, is there a reason why these textures are here? 

Should not go in assets?

**Review by JaimeTorrealba — CHANGES_REQUESTED** [review]

**Comment by JaimeTorrealba** [comment]
I check the example in netlify deploy, but I'm not sure if is right.

https://deploy-preview-116--cientos-tresjs.netlify.app/guide/staging/backdrop.html

image

File: public/models/suzanne.json
Comment (alvarosabu) [reply public/models/suzanne.json:1]: I though for docs would be ok to have them locally for performance, but seems like that's breaking when statically generating the assets.

**Pull request merged**